src/devices/cpu/z180/z180.cpp

```diff
@@ -1896,12 +1912,6 @@ void z180_device::device_reset()
 	m_frc_prescale = 0;
 
 	/* reset io registers */
-	m_asci_cntla[0] = (m_asci_cntla[0] & Z180_CNTLA0_MPBR_EFR) | Z180_CNTLA0_RTS0;
-	m_asci_cntla[1] = (m_asci_cntla[1] & Z180_CNTLA1_MPBR_EFR) | Z180_CNTLA1_CKA1D;
-	m_asci_cntlb[0] = (m_asci_cntlb[0] & (Z180_CNTLB0_MPBT | Z180_CNTLB0_CTS_PS)) | 0x07;
-	m_asci_cntlb[1] = (m_asci_cntlb[1] & Z180_CNTLB1_MPBT) | 0x07;
-	m_asci_stat[0] = m_asci_stat[0] & (Z180_STAT0_DCD0 | Z180_STAT0_TDRE);
-	m_asci_stat[1] = Z180_STAT1_TDRE;
```

Comment:
I don’t think this belongs in this changeset.

Reply:
Thanks. Missed this in reverting